src/main/scala/viper/silver/plugin/standard/inline/InlineErrorChecker.scala

```diff
@@ -37,8 +37,8 @@ trait InlineErrorChecker {
     val predicatesToInspect = predicateIds.map(program.findPredicate)
     val predicateCallGraph = PredicateCallGraph.graph(predicatesToInspect, program)
     val mutRecPreds = PredicateCallGraph.mutuallyRecursivePreds(predicateCallGraph)
-    mutRecPreds.foreach {
-      case Predicate(name, _, _) => println(s"Predicate: `$name` is mutually-recursive. Will not be inlined")
+    if (mutRecPreds.nonEmpty) {
+      prettyPrint(mutRecPreds, "mutually-recursive")
```

Comment:
I hate to be that guy but there's no need for the hyphen here lol
Also you're missing the final period in the error messages further down
But otherwise the actual code is good!!

Reply:
Merging after I address these